catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactory.java

```diff
@@ -120,7 +120,10 @@ public QueryRequest getQuery(GetRecordsType request) throws CswException {
 
         CswRecordMapperFilterVisitor filterVisitor = buildFilter(query.getConstraint());
         QueryImpl frameworkQuery = new QueryImpl(filterVisitor.getVisitedFilter());
-        frameworkQuery.setSortBy(buildSort(query.getSortBy()));
+
+        if (CswConstants.CSW_OUTPUT_SCHEMA.equals(request.getOutputSchema())) {
```

Comment:
Rather than check if this is the right output schema - I think it would be more precise to check if the AttributeRegistry.  The AttributeRegistry will contain all attributes available within the system. If it's not present we should probably return an error rather than fail quietly.